fleetctl/fleetctl.go

```diff
@@ -669,6 +669,81 @@ func createUnit(name string, uf *unit.UnitFile) (*schema.Unit, error) {
 	return &u, nil
 }
 
+// checkReplaceUnitState checks if the unit should be replaced
+// It takes a Unit object as a parameter
+// It returns 0 on success and if the unit should be replaced, 1 if the
+// unit should not be replaced; and any error acountered
+func checkReplaceUnitState(unit *schema.Unit) (int, error) {
```

Comment:
Please follow proper orthography: sentences always begin with a capital letter, and end in a full stop.

Note that this is particularly important for comments preceding functions (and other items), as godoc automatically treats them as doc comments, and tries to extract the first sentence for special handling.